docs/source/builtin_types.rst

```diff
@@ -22,19 +22,22 @@ Type                   Description
 ``Any``                dynamically typed value with an arbitrary type
 ====================== ===============================
 
-The type ``Any`` and type constructors such as ``List``, ``Dict``,
-``Iterable`` and ``Sequence`` are defined in the ``typing`` module.
+The type :py:data:`~typing.Any` and type constructors such as :py:class:`~typing.List`,
```

Comment:
I am thinking that all the types mentioned here are just too common to bother linking to their docs. I would only link the `typing` module itself.

Another idea: just above here is a table showing the most common built-in types. Maybe the items in the table should be linked instead?

Reply:
I thought about linking the table items instead, but the classes listed contain types, e.g. `Tuple[int, int]`. The `typing` module docs do not link those, e.g. [`Optional` docs](https://docs.python.org/3/library/typing.html#typing.Optional). I guess it counts as an example, which are not linked in the stdlib docs.